.coafile

```diff
@@ -34,6 +34,6 @@ max_lines_per_file = 1000
 [TODOS]
 enabled = False
 bears = KeywordBear
-
+language = python3
```

Comment:
Unrelated change to what this commit does? If so, this goes into its own commit.